ui-participant/src/api/api.tsx

```diff
@@ -286,7 +242,7 @@ export default {
   async submitPreRegResponse({ surveyStableId, surveyVersion, preRegResponse }:
                                {
                                  surveyStableId: string, surveyVersion: number,
-                                 preRegResponse: PreregistrationResponse
+                                 preRegResponse: Partial<PreregistrationResponse>
```

Comment:
Not great, but I don't think it's worse than the current cast of a partial PreregistrationResponse to a PreregistrationResponse.